backend/src/models/rentalAgency.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const mongoose = require('mongoose');
+
+// Define the rentalAgency schema
+
+const RentalAgency  = new mongoose.Schema({
+    name: {
+        type: String,
+        required: true
+    },
+    location: {
```

Comment:
Pull martins branche and define the location as him in the tour. There is special GeoJSON type for this.

Reply:
Defined location as GeoJSON type 'Point'